### ISSUE TITLE
Platforms attribute

### ISSUE THREAD

**Issue by johnthepink** [NewSpring/norma-meteor#8]
The platforms attribute in the `Norma` file doesn't seem to be making it's way in to the meteor app.

~~~
    platforms: [
      "android"
      "browser"
      "ios"
      "server"
    ]
~~~

This will results in a platforms file that is just:

~~~
browser
server
~~~

**Comment by jbaxleyiii** [comment]
@johnthepink we should revisit these methods: https://github.com/NewSpring/norma-meteor/blob/master/lib/manage.coffee#L44

And make them user Norma.config() and Norma.config.save()

**Comment by johnthepink** [comment]
Closed via #10
